test/e2e_test/YuvVideoSource.cc

```diff
@@ -46,31 +36,18 @@ EbErrorType YuvVideoSource::parse_file_info() {
         return EB_ErrorInsufficientResources;
     }
 
+    cal_yuv_plane_param();
+
     // Get file length
     fseek(file_handle_, 0, SEEK_END);
     file_length_ = ftell(file_handle_);
 
     // Seek to begin
     fseek(file_handle_, 0, SEEK_SET);
 
-    frame_length_ = width_ * height_;
-
-    // Calculate frame length
-    switch (image_format_) {
-    case IMG_FMT_420P10_PACKED:
-    case IMG_FMT_420: {
-        frame_length_ = frame_length_ * 3 / 2 * (bit_depth_ > 8 ? 2 : 1);
-    } break;
-    case IMG_FMT_422P10_PACKED:
-    case IMG_FMT_422: {
-        frame_length_ = frame_length_ * 2 * (bit_depth_ > 8 ? 2 : 1);
-    } break;
-    case IMG_FMT_444P10_PACKED:
-    case IMG_FMT_444: {
-        frame_length_ = frame_length_ * 3 * (bit_depth_ > 8 ? 2 : 1);
-    } break;
-    default: break;
-    }
+    uint32_t luma_size = width_ * height_ * bytes_per_sample_;
+    frame_length_ =
+        luma_size + ((luma_size >> (width_downsize_ + height_downsize_)) * 2);
 
```

Comment:
It should be keep consistent with Y4m Video Sources, the video info is printed out in Y4mVideo Sources.

Reply:
done, it is deleted.